src/Core/Commander/Providers/WMTS_Provider.js

```diff
@@ -318,48 +225,26 @@ define('Core/Commander/Providers/WMTS_Provider', [
         WMTS_Provider.prototype.executeCommand = function(command){
 
             //var service;
-            var destination = command.paramsFunction.layer.description.style.layerTile;
+            var destination = command.paramsFunction.destination;
             var tile = command.requester;
 
-            if(destination === 1)
-            {
-                return this.getColorTextures(tile,command.paramsFunction.layer.services).then(function(result)
-                {
+            if(destination === 1) {
```

Comment:
We should rename destination to something that makes more sense in this context. TextureType?

Test for textureType === "elevation" or textureType === "imagery" instead of 0/1.